src/styles/main.scss

```diff
@@ -1,3 +1,52 @@
-body {
-  margin: 0;
+@import "./variables";
+
+.page {
+  font-family: $font;
+  font-size: 30px;
+}
+
+.calendar {
+  box-sizing: border-box;
+  display: flex;
+  width: $cell-size * 7 + $gap-size * 6 + 20px;
+  padding: $translation-height 10px 10px;
+  flex-wrap: wrap;
+  gap: $gap-size;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    box-sizing: border-box;
+    width: $cell-size;
+    height: $cell-size;
+    border: 1px solid black;
+    list-style: none;
+    background-color: $cell-color;
+    transition: 500ms;
```

Comment:
If you use transition specify to what properties it should apply.
![image](https://github.com/mate-academy/layout_calendar/assets/63243136/4ea1b96c-e74f-4494-b67c-a686fd636975)